packages/cli/src/commands/generate/function/function.js

```diff
@@ -48,19 +48,65 @@ export const builder = (yargs) => {
       description: 'Name of the Function',
       type: 'string',
     })
+    .option('force', {
```

Comment:
This PR seems out-of-sync with the recent conversion to TS, which includes the options for either JS or TS language target. See:
https://github.com/redwoodjs/redwood/pull/1854/files#diff-9434dff1ad2a320fdf2411f1471ac4bc3a5b8202d59c8b4f5ac3bcd51b0c26a4

Do you just need to update from main?

Reply:
I had added back the "--force" option manually as I didn't realize it was already included. The rest of the file is the TS flavor/latest.